### ISSUE TITLE
unable to find the app version in the ios app

### ISSUE THREAD

**Issue by richlv** [mapillary/mapillary_issues#2638]
sorry if i have overlooked this, but totally unable to find the app version in the iphone app. "about" in settings only opens some webpage with general info.

**Comment by katrinhumal** [comment]
@richlv as for now, you can't check the version number in the app itself. You need to go to your phone settings and scroll down to Mapillary. We'll take note of this.

**Comment by richlv** [comment]
ah, thanks - that makes sense, although i expected that to be in the app menus somewhere :)

**Comment by katrinhumal** [comment]
That's a pretty normal expectation :) We'll try and see if we can find a nice place for it.

**Comment by katrinhumal** [comment]
@richlv sorry this has fallen between the chairs - totally my fault! It still stands that having the app version number visible in the app is a totally valid request. I've made sure this small task is waiting on the developer's desk when he returns from vacation in a few weeks. He said he'll be happy to implement in the next version he releases.

Closing the issue here as we're phasing out this repo.
